### DISCUSSION TITLE
How to identify bonds to get correct amount?

### DISCUSSION THREAD

**Discussion by emin63** [csingley/ofxtools#156] [Q&A]
Is there a way to identify that something is a bond (e.g., via CUSIP) to handle the amount correctly? For example, if I own $10,000 face of US Treasury Bonds, my brokers will often report it as a quantity of 10000 and a price of 101.25 or something like that. So if I multiple the quantity and price in the usual way, it will be 100x too large.

Currently I am just manually keeping track of which CUSIPs are bonds and dividing the quantity for those by 100. Is there a better or at least more automated way to handle this than me manually noting which CUSIPs have a units field which cannot be simply multiplied by the price?

Thanks.

**Comment by csingley** [comment]
The normal thing to do is to cross-reference the transaction with the SECINFO, using the pair of uniqueidtype/uniqueid (i.e. CUSIP) as a unique lookup key.  The SECINFO contains the asset class.

Does this sound like a job for sqlite?  You bet it does.
